circle.yml

```diff
@@ -15,10 +15,11 @@ dependencies:
     - "~/.cabal"
 
   override:
+    - rm -rf /home/ubuntu/.go_workspace/src
     - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
     - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
```

Comment:
Hey, this actually works, and is much more elegant! Thanks for the tip, great to have a devops-quality CI setup rather than what I was hacking together here!